data_steward/validation/main.py

```diff
@@ -130,31 +139,40 @@ def validate_all_hpos():
     """
     for item in resources.hpo_csv():
         hpo_id = item['hpo_id']
-        run_validation(hpo_id, True)
+        try:
+            run_validation(hpo_id, True)
```

Comment:
What happens if some other error happens? We want to continue no matter what happens yes?

Reply:
(Also a codacy issue) Remove the extra argument.